src/NzbDrone.Core/Notifications/Boxcar/Boxcar.cs

```diff
@@ -37,6 +39,14 @@ public override void OnRename(Series series)
         {
         }
 
+        public override void OnSystemUpdateAvailable(UpdatePackage package)
+        {
+            const string title = "New update is available";
+            var body = String.Format("New update is available - {0} - {1}", package.Version.ToString(), package.Url.ToString());
```

Comment:
If anything they may want to a link to access the Sonarr UI, but thats not really something we can provide (multiple IPs, not always accessible, etc).